go.mod

```diff
@@ -33,3 +34,5 @@ require (
 )
 
 replace github.com/aws/amazon-sagemaker-operator-for-k8s => github.com/aws/amazon-sagemaker-operator-for-k8s v1.0.1-0.20210303003444-0fb33b1fd49d
+
+replace github.com/flyteorg/flyteidl => github.com/kennyworkman/flyteidl v0.21.12-0.20211210005601-8c4bb8cda5fb
```

Comment:
just a reminder to update when idl PR merges..

Reply:
do you want me to resolve conflict and merge with my version of idl?